pkg/arvo/app/dojo.hoon

```diff
@@ -1390,8 +1408,10 @@
 ++  on-watch
   |=  =path
   ^-  (quip card:agent:gall _..on-init)
-  ~?  !=(our.hid src.hid)  [%dojo-peer-stranger src.hid]
-  ?>  (team:title our.hid src.hid)
+  ?>  ?|
+    (team:title our.hid src.hid)
+    (~(has in acl) src.hid)
+    ==
```

Comment:
wasn't sure how to format this. Should probably restore the `%dojo-peer-stranger` printf too.

Reply:
This is canonical

```
?>  ?|  (team:title our.hid src.hid)
        (~(has in acl) src.hid)
    ==
```